framework/src/main/java/org/checkerframework/common/wholeprograminference/WholeProgramInferenceScenes.java

```diff
@@ -835,6 +868,20 @@ public AnnotatedTypeMirror atmFromATypeElement(TypeMirror typeMirror, ATypeEleme
         return storage.atmFromATypeElement(typeMirror, type);
     }
 
+    /**
+     * Obtain the type from an AnnotatedTypeMirror.
+     *
+     * @param typeMirror the underlying type for the result
+     * @param type the AnnotatedTypeMirror from which to obtain annotations
+     * @return an annotated type mirror with underlying type {@code typeMirror} and annotations from
+     *     {@code type}
+     */
+    public AnnotatedTypeMirror atmFromAnnotatedTypeMirror(
+            TypeMirror typeMirror, AnnotatedTypeMirror type) {
+        // return storage.atmFromAnnotatedTypeMirror(typeMirror, type);
+        throw new BugInCF("Not used in WholeProgramInferenceJavaParser");
```

Comment:
Should this method be deleted?

Reply:
Yes.  Thanks for catching that.  It may be reinstated in a future refactor.